config/application.rb

```diff
@@ -26,6 +26,8 @@ class Application < Rails::Application
     # Skip views, helpers and assets when generating a new resource.
     config.api_only = true
 
+    # config.active_job.queue_adapter = :sidekiq
```

Comment:
That's to determine which backend to use for the queue implementation. sidekiq a worker/queue that backs onto redis. There are also synchronous queues, db backed queues, etc. I'm still thinking about which queue implementation to use.